internal/template/templates/.claude/skills/moai-workflow-project/scripts/navigator-audit.sh

```
@@ -0,0 +1,593 @@
+    mv -f "$tmp" "$dest" 2>/dev/null || cp "$tmp" "$dest" 2>/dev/null || true
+    rm -f "$tmp" 2>/dev/null || true
+}
+
+# normalize <string>: lowercase, replace non-alphanumeric runs with single
+# hyphen, collapse, strip leading/trailing hyphens.
+normalize() {
+    printf '%s' "$1" | tr '[:upper:]' '[:lower:]' \
+        | tr -c 'a-z0-9\n' '-' \
+        | tr -s '-' '-' \
+        | sed 's/^-*//; s/-*$//'
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)